modules/developer_manual/pages/core/theming.adoc

```diff
@@ -149,33 +174,20 @@ And here’s a longer, more complete example:
 </info>
 ----
 
-The value of the `id` element needs to be the name of your theme’s
-folder. We recommend that it always be prefixed with `theme-`. The main
-reason for doing so, is that it is alphabetically sorted in a terminal
+The value of the `id` element needs to be the name of your theme’s folder. We recommend that it always be prefixed with `theme-`. The main reason for doing so, is that it is alphabetically sorted in a terminal
 when handling app folders.
 
-The `type` element needs to be the same as is listed above, so that
-ownCloud knows to handle the app as a theme. The dependencies element
-needs to be present to set the minimum and maximum versions of ownCloud
-which are supported. If it’s not present, a warning will be displayed in
-ownCloud 10 and an error will be thrown in the upcoming ownCloud 11.
+The `type` element needs to be the same as is listed above, so that ownCloud knows to handle the app as a theme. The dependencies element needs to be present to set the minimum and maximum versions of ownCloud which are supported. If it’s not present, a warning will be displayed in ownCloud 10 and an error will be thrown in the upcoming ownCloud 11.
 
-While the remaining elements are optional, they help when working with
-the theme in the ownCloud Admin dashboard. Please consider filling out
-as many as possible, as completely as possible.
+While the remaining elements are optional, they help when working with the theme in the ownCloud Admin dashboard. Please consider filling out as many as possible, as completely as possible.
 
 === Theme Signing
 
-If you are going to publish the theme as an app in the
-{oc-marketplace-url}[marketplace], you need to sign it.
-However, if you are only creating a private theme for your own ownCloud
-installation, then you do not need to.
+If you are going to publish the theme as an app in the {oc-marketplace-url}[marketplace], you need to sign it. However, if you are only creating a private theme for your own ownCloud installation, then you do not need to.
 
-That said, to avoid a signature warning in the ownCloud UI, you need to
-add it to the `integrity.ignore.missing.app.signature` list in
-`config/config.php`. The following example allows the app whose
-application id is `app-id` to have no signature.
+That said, to avoid a signature warning in the ownCloud UI, you need to add it to the `integrity.ignore.missing.app.signature` list in `config/config.php`. The following example allows the app whose application id is `app-id` to have no signature.
```

Comment:
```suggestion
That said, to avoid a signature warning in the ownCloud UI, you need to add it to the `integrity.ignore.missing.app.signature` list in `config/config.php`. The following example allows the app whose application ID is `app-id` to have no signature.
```